ctapipe_io_lst/containers.py

```diff
@@ -136,15 +135,14 @@ class LSTContainer(Container):
     """
     Storage for the LSTCameraContainer for each telescope
     """
-    tels_with_data = Field([], "list of telescopes with data")
 
     # create the camera container
     tel = Field(
         Map(LSTCameraContainer),
         "map of tel_id to LSTTelContainer")
 
 
-class LSTDataContainer(DataContainer):
+class LSTArrayEventContainer(ArrayEventContainer):
```

Comment:
Note that the ctapipe `ArrayEventContainer` already contains a "mon" field - so the one here is not needed, only the *lst* field

Reply:
good catch, thanks